tests/models/qwen2_vl/test_modeling_qwen2_vl.py

```diff
@@ -166,6 +166,8 @@ def prepare_config_and_inputs_for_common(self):
         input_ids = ids_tensor([self.batch_size, self.seq_length], self.vocab_size)
         attention_mask = torch.ones(input_ids.shape, dtype=torch.long, device=torch_device)
 
+        input_ids[:, -1] = self.pad_token_id
```

Comment:
for my understanding, any reason why last token has to be a pad?

Reply:
to avoid index error 

(modeling code)
> vision_tokens = input_ids[vision_start_indices + 1]